src/adapter/graphql/src/scalars/odf_generated.rs

```diff
@@ -135,8 +135,8 @@ pub struct Checkpoint {
     pub size: u64,
 }
 
-impl From<odf::Checkpoint> for Checkpoint {
-    fn from(v: odf::Checkpoint) -> Self {
+impl From<odf::metadata::Checkpoint> for Checkpoint {
+    fn from(v: odf::metadata::Checkpoint) -> Self {
```

Comment:
Comment for readers:
`odf::Checkpoint` & `odf::metadata::Checkpoint` are the same structure